.changeset/good-camels-pull.md

```diff
@@ -0,0 +1,8 @@
+---
+'@astrojs/markdoc': minor
+'@astrojs/mdx': major
+'@astrojs/markdown-remark': major
+'astro': major
+---
+
+TODO:
```

Comment:
I guess this is waiting on feedback in https://github.com/withastro/docs/pull/12510 but we should be sure to update it before merging!